docs/source/openstack/volume-migration-using-s3.md

```diff
@@ -0,0 +1,190 @@
++------------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+In both cases, record the `access` and `secret` values, because we're going to need those in a subsequent step.
+
+## Find object store endpoints
+
+Look up information on the `object-store` service in each environment. We need to know the base URL for this service.
+
+Information from Kaizen:
+
+```
+$ openstack --os-cloud kaizen catalog show object-store -c endpoints
++-----------+-------------------------------------------------------+
+| Field     | Value                                                 |
++-----------+-------------------------------------------------------+
+| endpoints | moc-kzn                                               |
+|           |   admin: https://kzn-swift.massopen.cloud/swift/v1    |
+|           | moc-kzn                                               |
+|           |   public: https://kzn-swift.massopen.cloud/swift/v1   |
+|           | moc-kzn                                               |
+|           |   internal: https://kzn-swift.massopen.cloud/swift/v1 |
+|           |                                                       |
++-----------+-------------------------------------------------------+
+```
+
+And from the NERC:
+
+```
+$ openstack --os-cloud nerc catalog show object-store -c endpoints
++-----------+----------------------------------------------------------------------------------------+
+| Field     | Value                                                                                  |
++-----------+----------------------------------------------------------------------------------------+
+| endpoints | regionOne                                                                              |
+|           |   admin: http://10.255.2.253:8080                                                      |
+|           | regionOne                                                                              |
+|           |   internal: http://10.255.2.253:8080/v1/AUTH_01234567890123456789012345689012          |
+|           | regionOne                                                                              |
+|           |   public: https://stack.nerc.mghpcc.org:13808/v1/AUTH_01234567890123456789012345689012 |
+|           |                                                                                        |
++-----------+----------------------------------------------------------------------------------------+
+```
+
+## Configure minio client
+
+For each environment, create a MinIO alias configuration using the base URL of the "public" interface of the object-store service and the access key and secret your generated earlier:
+
+```
+$ mc alias set kaizen https://kzn-swift.massopen.cloud 1234567890ABCDEF1234567890ABCDEF 1234567890ABCDEF1234567890ABCDEF
```

Comment:
same comment about path instead of just `mc`

Reply:
Naved brought up a good point, if we have them create a linux virtual machine to do this work from we will not run into issues where they are on windows or mac.